letsencrypt/scripts/main.py

```diff
@@ -122,91 +135,57 @@ def main():  # pylint: disable=too-many-branches
     if not args.eula:
         display_eula()
 
-    # Make sure we actually get an installer that is functioning properly
-    # before we begin to try to use it.
-    try:
-        installer = client.determine_installer(config)
-    except errors.LetsEncryptMisconfigurationError as err:
-        logging.fatal("Please fix your configuration before proceeding.  "
-                      "The Installer exited with the following message: "
-                      "%s", err)
-        sys.exit(1)
+    # list of (Description, Known Authenticator classes, init arguments)
+    all_auths = [
+        ("Apache Web Server", configurator.ApacheConfigurator, config),
+        ("Standalone Authenticator", standalone.StandaloneAuthenticator),
```

Comment:
On top of creating instances here https://github.com/letsencrypt/lets-encrypt-preview/pull/250/files#r25096997, let's make the description available as class instance, e.g. `StandaloneAuthenticator.description == "Standalone Authenticator"`, etc.